### PR TITLE
Adding new product card component

### PR DESCRIPTION
**Pull request by thenose-samuel**
Hello, I'm adding a new component called product card

### CODE REVIEW

File: components/product-card.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<div class="flex flex-row justify-center w-screen h-screen place-items-center">
+    <div class="h-48 duration-500 bg-white rounded-md group w-52 hover:h-72 hover:shadow-2xl tranform">
+      <div class="h-48">
+        <img class="object-cover cursor-pointer rounded-t-md" src="https://cdn.pixabay.com/photo/2016/02/29/00/19/cake-1227842_1280.jpg" alt="product-image" />
```
Comment (bobbyiliev) [inline components/product-card.html]: Thank you for the PR! A few remarks:

Can you change the image with an image from https://unsplash.com/?

File: components/product-card.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          <div class="w-4 h-4 bg-green-600 rounded-full"></div>
+          <div class="w-4 h-4 bg-purple-600 rounded-full"></div>
+        </div>
+        <div class="flex flex-row justify-between mt-4">
+          <svg xmlns="http://www.w3.org/2000/svg" class="w-5 h-5 cursor-pointer" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+            <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4.318 6.318a4.5 4.5 0 000 6.364L12 20.364l7.682-7.682a4.5 4.5 0 00-6.364-6.364L12 7.636l-1.318-1.318a4.5 4.5 0 00-6.364 0z" />
+          </svg>
+          <svg xmlns="http://www.w3.org/2000/svg" class="w-5 h-5 cursor-pointer" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+            <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M3 3h2l.4 2M7 13h10l4-8H5.4M7 13L5.4 5M7 13l-2.293 2.293c-.63.63-.184 1.707.707 1.707H17m0 0a2 2 0 100 4 2 2 0 000-4zm-8 2a2 2 0 11-4 0 2 2 0 014 0z" />
+          </svg>
```
Comment (bobbyiliev) [inline components/product-card.html]: There seem to be extra spaces on each line. Could you adjust that so that the formatting is consistent? Thank you!

**Review by bobbyiliev — CHANGES_REQUESTED** [review]

File: components/product-card.html (hunk 1 above)
Comment (thenose-samuel) [reply components/product-card.html]: Thanks for the review. Yes I shall make the change

File: components/product-card.html (hunk 2 above)
Comment (thenose-samuel) [reply components/product-card.html]: Sure I will do that

**Review by bobbyiliev — APPROVED** [review]

**Pull request merged**